dspace-spring-rest/src/main/java/org/dspace/app/rest/model/SelectableMetadata.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+/**
+ * The SelectableMetadata REST Resource. It is not addressable directly, only
+ * used as inline object in the InputForm resource
```

Comment:
These are the same values that can be retrieved from the authority endpoint, correct? Do we have to include them in the InputForm resource?

Reply:
SelectableMetadata is an inner resource in the inputform resource, it is different from the values retrieved into authority endpoint.

Reply:
I thought every value-pair was also auto-registered as an authority now? Can you give an example of a use case where this doesn't link to a value-pair?

Reply:
SelectableMetadata was introduced to make a clear distinction between the cases where a value-pairs was used as an authority list of acceptable values (dropdown) and where it was used to allow to pick the metadata to use to store the value (qualdrop_values).
If a value-pair is used by a qualdrop_value it is not autoregistered as an authority see https://github.com/DSpace/DSpace/pull/1852/files/779f561e6890c9e83e70456213fa0ced41f1fb07#diff-3b9150885fcc1931b62efc9ddd18c9aeR292 it is exposed as an array of SelectableMetadata object

Reply:
Ok that clarifies it. But I would add that to the comments of this class.